charts/camunda-platform-8.5/test/unit/zeebe/configmap_test.go

```diff
@@ -147,3 +150,42 @@ func (s *ConfigmapTemplateTest) TestDifferentValuesInputs() {
 
 	testhelpers.RunTestCasesE(s.T(), s.chartPath, s.release, s.namespace, s.templates, testCases)
 }
+
+func (s *ConfigmapTemplateTest) TestStartupScriptExecsPresentInConfigmap() {
+	// given
+	options := &helm.Options{
+		KubectlOptions: k8s.NewKubectlOptions("", "", s.namespace),
+	}
+
+	// Fetch the startup script from upstream repository
```

Comment:
I really like the idea here, but it doesn't fit the unit test.
The best practice for unit testing is to use static data.

Given that it's not something that always changes, let's use the string directly.
We have planned tasks to test things dynamically in the integration tests.